health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -109,44 +110,288 @@ const BoundaryRelationCreate = () => {
+                params: {},
+                body: {
+                  SearchCriteria: {
+                    id: [id],
+                    tenantId: tenantId,
+                    type: typeOfData,
+                  },
+                },
+              });
+      
+              const status = searchResponse?.ResourceDetails?.[0]?.status;
+              let errorString = searchResponse?.ResourceDetails?.[0]?.additionalDetails.error;
```

Biome

[error] This let declares a variable that is only assigned once.

'errorString' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)